### PR TITLE
Add Sept blog post

### PR DESCRIPTION
**Pull request by d-m-u**
🎉

### CODE REVIEW

File: site/_posts/2018-10-02-last-month-in-miq-building-a-blog-post.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+[UI appliances without the reporting role can create queue messages that never get processed.](https://github.com/ManageIQ/manageiq/pull/18041)
+
+[Encrypted values must be encrypted twice.](https://github.com/ManageIQ/manageiq/pull/18031)
+
+[AR models do not need to be backed by a canonical table.](https://github.com/ManageIQ/manageiq/pull/18003)
+
+[It is possible to queue things for a zone that doesn't exist in the region in question.](https://github.com/ManageIQ/manageiq/pull/17987)
+
+All misspellings are actually Britishisms in disguise.
```
Comment (jrafanie) [inline site/_posts/2018-10-02-last-month-in-miq-building-a-blog-post.md]: 🇺🇸  > 🇬🇧 

:trollface: @juliancheal

Comment (juliancheal) [reply site/_posts/2018-10-02-last-month-in-miq-building-a-blog-post.md]: @jrafanie It's nice to feel included.

**Comment by JPrause** [comment]
LTGM! :shipit:

**Comment by JPrause** [comment]
@cybette would you be able to review/merge.

**Pull request merged**